packages/tests/README.md

```diff
@@ -0,0 +1,234 @@
+# Tests
```

Comment:
Thank you for adding, this!

Reply:
then we'll keep it!

Reply:
FYI we already have some testing documentation at https://github.com/lit/lit/blob/main/CONTRIBUTING.md#tests. Would be nice to unify.